psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -1045,9 +1049,9 @@ void ESPPropCalc::compute_esp_over_grid(bool print_output) {
     std::shared_ptr<ElectrostaticInt> epot(dynamic_cast<ElectrostaticInt*>(integral_->electrostatic().release()));
 
     if (print_output) {
-        outfile->Printf("\n Electrostatic potential computed on the grid and written to grid_esp.dat\n");
+        outfile->Printf("\n Electrostatic potential to be computed on the grid and written to grid_esp.dat\n");
     }
-
+        
```

Comment:
```suggestion

```

Reply:
@JonathonMisiewicz I accidentally marked this as resolved, please let me know if additional changes are needed for approval of this PR.